### ISSUE TITLE
Error 500 installing XERA

### ISSUE THREAD

**Issue by mxmarva** [mahtab2003/Xera#188]
Hello, I am getting a 500 error when trying to install XERA.

As soon as I access /install.php I get error 500.

By default my server uses PHP 8.2, I already lowered it to 8.1 and the problem persists, with versions prior to 8.0 the message appears that Composer requires 8.1 or higher.

I don't know if you can advise me.

**Comment by itsmerosu** [comment]
Have you tried using PHP version 7.4?

**Comment by itsmerosu** [comment]
> Hello, I am getting a 500 error when trying to install XERA.
> 
> As soon as I access /install.php I get error 500.
> 
> By default my server uses PHP 8.2, I already lowered it to 8.1 and the problem persists, with versions prior to 8.0 the message appears that Composer requires 8.1 or higher.
> 
> I don't know if you can advise me.

Have you tried using PHP version 7.4? Please mention a screenshot of the error.log.

**Comment by mxmarva** [comment]
Yes, it was indeed a solution I saw in another thread, but it tells me that Compose requires PHP 8.1 or higher.

**Comment by mahtab2003** [comment]
Closed due to inactivity
